langlib/lang.error/src/main/ballerina/error.bal

```diff
@@ -42,6 +47,13 @@ public isolated function message(error e) returns string = @java:Method {
 
 # Returns the error's cause.
```

Comment:
There can be errors without cause ryt?. We need to mention that.

Reply:
This will have to come from the spec, didn't change any of the actual documentation here.

Reply:
https://github.com/ballerina-platform/ballerina-spec/issues/1199